node/impl/full/eth.go

```diff
@@ -175,6 +175,7 @@ type EthAPI struct {
 
 	EthModuleAPI
 	EthEventAPI
+	StateAPI StateAPI
```

Comment:
what do we need this for?

Reply:
Oh, it was left to be removed, in my previous implementation it was being used for MsgLookup which is not required anymore. I'll remove it